frontend/src/i18n/locales/de.yaml

```diff
@@ -6,3 +6,22 @@ navigation: Navigation
 page-not-found:
   title: Seite nicht gefunden
   body: Die angefragte Resource existiert nicht. Sorry! :-(
+
+about:
+  title: Information
+  body: >
+    Tobira ist ein Videoportal für Opencast. Es erlaubt seinen Nutzern, alle
+    Episoden und Serien aus der verbundenen Opencast-Instanz zu durchstöbern,
```

Comment:
Should we say "aus **den** verbundenen Opencast-Instanzen"? We already talked about potentially having more than one, right? :thinking:

Reply:
The text will likely still change a ton anyway, so don't really consider it "final". I would keep it like this for now, don't wanna make promises :D

Reply:
Makes sense.